tests/lint/pylintrc

```diff
@@ -80,8 +80,13 @@ disable=design,
         useless-object-inheritance,
         no-else-return,
         len-as-condition,
-        cyclic-import,       # disabled due to the inevitable dgl.graph -> dgl.subgraph loop
-        undefined-variable,  # disabled due to C extension (should enable)
+        cyclic-import,          # disabled due to the inevitable dgl.graph -> dgl.subgraph loop
+        undefined-variable,     # disabled due to C extension (should enable)
+        raise-missing-from,     # meh
```

Comment:
What's this comment?

Reply:
https://www.urbandictionary.com/define.php?term=Meh